tests/src/test/java/alluxio/master/MasterHealthCheckClientIntegrationTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master;
+
+import alluxio.BaseIntegrationTest;
+import alluxio.HealthCheckClient;
+import alluxio.LocalAlluxioClusterResource;
+import alluxio.PropertyKey;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class MasterHealthCheckClientIntegrationTest extends BaseIntegrationTest {
+
+  @Rule
+  public LocalAlluxioClusterResource mLocalAlluxioClusterResource =
+          new LocalAlluxioClusterResource.Builder()
+                  .setProperty(PropertyKey.USER_RPC_RETRY_MAX_NUM_RETRY, 30).build();
+
+  private LocalAlluxioCluster mLocalAlluxioCluster = null;
+  private HealthCheckClient mHealthCheckClient;
+
+  @Before
+  public final void before() throws Exception {
+    mLocalAlluxioCluster = mLocalAlluxioClusterResource.get();
+    mHealthCheckClient = new MasterHealthCheckClient();
+  }
+
+  @After
+  public final void after() throws Exception {
+    mLocalAlluxioCluster.stop();
```

Comment:
the `LocalAlluxioClusterResource` rule will already clean up the cluster at the end of the test

Reply:
Done